pandas/_config/config.py

```diff
@@ -51,15 +51,16 @@
 from collections import namedtuple
 from contextlib import contextmanager
 import re
+from typing import Dict
 import warnings
 
 DeprecatedOption = namedtuple('DeprecatedOption', 'key msg rkey removal_ver')
 RegisteredOption = namedtuple('RegisteredOption',
                               'key defval doc validator cb')
 
-_deprecated_options = {}  # holds deprecated option metdata
```

Comment:
can you leave all of the comments, you can put them above each option

Reply:
no problem.

Reply:
Alright, its done. Is it ok?